src/main/java/reposense/frontend/CliArguments.java

```diff
@@ -19,7 +20,7 @@
     private static final String SINCE_DATE_ARG = "since";
     private static final String UNTIL_DATE_ARG = "until";
 
-    private static final String PARSE_EXCEPTION_MESSAGE_INVALID_INPUTS = "Failed to parse inputs arguments";
+    private static final String MESSAGE_INVALID_INPUTS = "Failed to parse inputs arguments";
     private static final String PARSE_EXCEPTION_MESSAGE_NO_CSV_FILE = "Failed due to missing CSV file";
```

Comment:
As mentioned, normally when we comment on one, it applies to all.

Reply:
👍 Will take note in the future.